supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc

```diff
@@ -33,21 +33,18 @@
 // RHV: Added "In Red Hat Virtualization," and added blank line so that first bullet rendered
 [[header-bar]]
 ==== image:images/yes.png[yes] header bar (noun)
-*Description*: In Red Hat Virtualization, the _header bar_ contains the following buttons:
+*Description*: In the Red Hat Virtualization VM Portal, the _header bar_ contains the following buttons:
```

Comment:
```suggestion
*Description*: In the Red Hat Virtualization VM Portal, the _header bar_ contains buttons to help you to manage your display, including notifications and account settings.
```

Suggesting a generic sentence, you might want to reword the actual text of my suggested sentence.